graalpy/graalpy-javase-guide/README.md

```diff
@@ -0,0 +1,257 @@
+# Using Python Packages in a Java SE Application
+
+Python libraries can be used in and shipped with plain Java applications.
+The [GraalPy Maven artifacts](https://central.sonatype.com/artifact/org.graalvm.polyglot/python) and [GraalVM Polyglot APIs](https://www.graalvm.org/latest/reference-manual/embed-languages/) allow flexible integration with different project setups.
+
+Using Python packages in Java projects often requires a bit more setup, due to the nature of the Python packaging ecosystem.
+GraalPy provides a [python-embedding](https://central.sonatype.com/artifact/org.graalvm.python/python-embedding) package that simplifies the required setup to ship Python packages as Java resources or in separate folders.
+The important entry points to do so are the [VirtualFileSystem](https://github.com/oracle/graalpython/blob/master/docs/user/Embedding-Build-Tools.md#virtual-filesystem) and the [GraalPyResources](https://github.com/oracle/graalpython/blob/master/docs/user/Embedding-Build-Tools.md#deployment) classes.
+
+## 1. Getting Started
+
+In this guide, we will add a small Python library to [generate QR codes](https://pypi.org/project/qrcode) to a Java GUI application:
+![Screenshot of the app](screenshot.png)
+
+## 2. What you will need
+
+To complete this guide, you will need the following:
+
+ * Some time on your hands
+ * A decent text editor or IDE
+ * A suppported JDK[^1], preferably the latest [GraalVM JDK](https://graalvm.org/downloads/) or newer
+
+ [^1]: JDK 17 is supported with interpreter only, JDK 21 and newer with JIT compilation.
```

Comment:
I still think we should warn people against trying with GraalVM for JDK 17. Maybe something like:
```suggestion
 [^1]: Oracle/Open/GraalVM JDK 21 or later are supported with JIT compilation, Oracle/Open JDK 17 are supported with interpreter only. GraalVM for JDK 17 is not supported.
```

Reply:
I took the liberty and applied similar wording